tests/testthat/test-sce_to_anndata.R

```diff
@@ -20,7 +23,23 @@ test_that("Conversion of SCE to AnnData works as expected", {
   expect_equal(colnames(colData(sce)), colnames(colData(converted_sce)))
   expect_equal(colnames(rowData(sce)), colnames(rowData(converted_sce)))
 
+  # test that H5 file is created with new assay name
+  # add logcounts
+  logcounts(sce) <- counts(sce)
+  new_anndata_file <- "test_anndata_logcounts.h5"
+  expect_snapshot_file({
```

Comment:
This test is probably fine for most cases, but we could reload the file and check its contents as well for a more complete test. Since we already have the package loaded, I would reload the file with `zellconverter::AnnData2SCE(new_anndata_file)` directly and check the contents of the `X` assay are equal to `logcounts(sce)`.

Reply:
I thought about this, but when you read the object back in it just uses the stored name for the `X_assay` so the result is an sce object with both `logcounts` and `counts` as assays, regardless of what is stored in `X`.

Reply:
what happens when you use `zellconverter::AnnData2SCE(new_anndata_file, X_name = "X")`?

Reply:
I used `zellkonverter::readH5AD` since `AnnData2SCE` requires `reticulate` and an `AnnData` object actually stored in R. That function also has the `X_name` argument, so I used that to add these tests in https://github.com/AlexsLemonade/scpcaTools/pull/219/commits/30f2569a06451d417976f5c613ed2a3d44e8b5c3